CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## Important Notes
 
+- [#895](https://github.com/oauth2-proxy/oauth2-rpoxy/pull/895)/[#911](https://github.com/oauth2-proxy/oauth2-rpoxy/pull/911) Validate and log provider type on startup.
```

Comment:
We only link to the PR normally, any additional discussion is normally linked from there
```suggestion
- [#911](https://github.com/oauth2-proxy/oauth2-proxy/pull/911) Validate and log provider type on startup.
```

Reply:
Will fix.

Reply:
Fixed.